File: client/src/app/model/ClientRecord.ts
export class ClientRecord {
  public id: string;
  public surname: string;
  public name: string;
  public age: number;

  assign(a: any) {
    this.id = a.id;
    this.surname = a.surname;
    this.name = a.name;
    this.age = a.age;
  }

  static of(a: any): ClientRecord {
    let ret = new ClientRecord();
    ret.assign(a);
    return ret;
  }
}

File: controller/src/kz/pompei/springAngular/controller/model/ClientRecord.java
package kz.pompei.springAngular.controller.model;

public class ClientRecord {
  public String id;
  public String surname;
  public String name;
  public int age;

  public ClientRecord() {}

  public ClientRecord(String id, String surname, String name, int age) {
    this.id = id;
    this.surname = surname;
    this.name = name;
    this.age = age;
  }
}
